packages/gis/lua/Layer.lua

```diff
@@ -889,6 +915,80 @@ Layer_ = {
 	-- print(bbox.xMin, bbox.yMin, bbox.xMax, bbox.yMax)
 	box = function(self)
 		return TerraLib().getBoundingBox(self.project.layers[self.name])
+	end,
+	--- Creates a vector data from a raster layer using polygons that have an homogeneous internal content.
```

Comment:
...using polygons covering contiguous pixels that share the same value.

Reply:
Indicate that the created layer will have an attribute called `value` with the value of the pixel.